Sources/Transifex/Core.swift

```diff
@@ -298,10 +301,17 @@ class NativeCore : TranslationProvider {
 Error rendering source string '\(sourceString)' with string to render '\(stringToRender)'
  locale code: \(localeCode) params: \(params). Error: \(error)
 """)
-            return errorPolicy.get(sourceString: sourceString,
-                                   stringToRender: stringToRender,
-                                   localeCode: localeCode,
-                                   params: params)
+            do {
+                return try errorPolicy.get(sourceString: sourceString,
+                                           stringToRender: stringToRender,
+                                           localeCode: localeCode,
+                                           params: params)
+            }
+            catch {
+                Logger.error("Error running error policy: \(error)")
```

Comment:
How about we also include `sourceString` in the log?

Reply:
Great idea, updated the log so that it contains all of the information (source string, string to render, locale code and params).